internal/service/s3/bucket_test.go

```diff
@@ -2507,7 +2507,7 @@ func TestAccS3Bucket_Replication_RTC_valid(t *testing.T) {
 
 func TestAccS3Bucket_Manage_objectLock(t *testing.T) {
```

Comment:
Tests with configurable object lock configuration arguments have been ported over to #22644